src/pybind/mgr/rgw/module.py

```diff
@@ -166,10 +277,10 @@ def _cmd_rgw_zone_create(self,
 
     @CLICommand('rgw realm reconcile', perm='rw')
     def _cmd_rgw_realm_reconcile(self,
-                             realm_name : Optional[str] = None,
-                             zonegroup_name: Optional[str] = None,
-                             zone_name: Optional[str] = None,
-                             update: Optional[bool] = False):
+                                 realm_name: Optional[str] = None,
+                                 zonegroup_name: Optional[str] = None,
+                                 zone_name: Optional[str] = None,
+                                 update: Optional[bool] = False):
         """Bootstrap new rgw zone that syncs with existing zone"""
```

Comment:
What does realm_reconcile do? Does it update zone's endpoints?

Reply:
This is legacy code. I'm planning to remove these cmds from the docs as no body are using them nor they are really well tested to have them documented (so some user can go and user them). WDYT?

Reply:
If you don't see any use for it, I think it's fair game to remove this code.

Reply:
By now I'll just remove the cmds from the docs keeping the code so users don't make use of these commands.